dist/gmusic.js

```diff
@@ -222,16 +272,23 @@ proto.playback = {
       return GMusic.Playback.NO_REPEAT;
     }
   },
-
-  toggleRepeat: function (mode) {
-    if (!mode) {
-      // Toggle between repeat modes once
-      this.playback._repeatEl.click();
-    } else {
-      // Toggle between repeat modes until the desired mode is activated
+  setRepeat: function (mode) {
+    if (mode === GMusic.Playback.SINGLE_REPEAT ||
+        mode === GMusic.Playback.LIST_REPEAT ||
+        mode === GMusic.Playback.NO_REPEAT) {
       while (this.playback.getRepeat() !== mode) {
-        this.playback._repeatEl.click();
+        this.playback.toggleRepeat();
       }
+    } else {
+      console.warn('Invalid repeat mode supplied');
+    }
+  },
+  toggleRepeat: function () {
+    if (arguments.length > 0) {
+      console.warn('toggleRepeat mode argument has been deprecated. Use setRepeat instead.');
+      this.playback.setRepeat.call(GMusic, arguments);
```

Comment:
I think you mean `apply` here. However we also know that there's only 1 parameter so we could use `mode` directly instead of `call/apply`